Dockerfile

```diff
@@ -44,11 +44,13 @@ RUN cd $VECTOR_ARCHIVE && ./install.sh -express $II_SYSTEM VW -noad && hostname
 RUN cp $II_SYSTEM/ingres/.ingVWsh /etc/profile.d/vectorVW.sh
 
 # fix hostname as localhost
-RUN source /etc/profile.d/vectorVW.sh && sed -i -e "s,`cat /tmp/hostname.build`,localhost,gI" $II_SYSTEM/ingres/files/config.dat && ingsetenv II_HOSTNAME localhost
+RUN source /etc/profile.d/vectorVW.sh && sed -isaved -e "s,`cat /tmp/hostname.build`,localhost,gI" -e 's/mgmtsvr:.*/mgmtsvr: 0/' $II_SYSTEM/ingres/files/config.dat && ingsetenv II_HOSTNAME localhost
```

Comment:
I do not understand, why this change is required.

Reply:
Which one?  ;-)
-isaved causes the in-place edit by sed to retain a backup file of the config.dat just because that seems like a good idea, especially in diagnostic or debugging.
The management server is a massive component and should not be included in a container.  I was attempting to make the container at least slightly more svelte.....

Reply:
I am not familiar with all Ingres components, but does the Vector container work without the mgmtsvr?

Reply:
Yes, Vector will work without mgmtsvr - its only used by the Director (client).

Should someone want to add/restore this they can do so by tweaking the Dockerfile (i.e. not worth making a parameter).

I'm in favor of this

Reply:
A better long term solution would be to make the management server a stand alone container but yes, it is easy for someone to modify the Dockerfile if that was an issue. (There is no scenario I can think of where someone would do this ;-)